docs/source/technical_tutorials/access/accessing_gedi_l4a_data.ipynb

```diff
@@ -2,20 +2,20 @@
  "cells": [
```

Comment:
<p>I tried to click on the 'Searching for Granules in MAAP' https://docs.maap-project.org/en/latest/search/granules.html but I get a 404 not found</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
@smk0033

Reply:
<p>I believe this is what it should point to https://docs.maap-project.org/en/latest/technical_tutorials/search/granules.html</p><p>Does that look right?</p>

Reply:
Yup! I think I have it updated to that in a new PR for the accessing_gedi_l3 tutorial, and I'll have it updated to that as well soon for the gedi_l4a tutorial. I wasn't sure if I should start pushing individual tutorials until we get this PR merged